packages/block-editor/src/components/block-patterns-list/index.js

```diff
@@ -101,6 +102,21 @@ function BlockPattern( {
 									{ pattern.description }
 								</VisuallyHidden>
 							) }
+							{ pattern.id && ! pattern.syncStatus && (
+								<Tooltip
+									position="top center"
+									text={ __(
+										'Editing this pattern will also update anywhere it is used'
```

Comment:
This icon is not tabbable and thus won't announce this text for screen readers. Maybe we should merge this into the description above? (And yep probably it's also an issue in the patterns page in the site editor? 😅 )

Reply:
I wonder if it would be more annoying than useful to hit that for every pattern, especially if tabbing through a filtered list of just synced patterns? Do you think we could look at this as a follow up and cover site editor and post inserter together?

Reply:
Yes I meant as a follow-up to look into in the future!